addons/website_sale/models/res_config_settings.py

```diff
@@ -30,6 +30,12 @@ class ResConfigSettings(models.TransientModel):
     cart_abandoned_delay = fields.Float("Abandoned Delay", help="Number of hours after which the cart is considered abandoned.",
                                         related='website_id.cart_abandoned_delay', readonly=False)
     cart_add_on_page = fields.Boolean("Stay on page after adding to cart", related='website_id.cart_add_on_page', readonly=False)
+    terms_url = fields.Char(compute='_compute_terms_url', string="URL", help="A preview will be available at this URL.")
+
+    @api.depends('website_id')
```

Comment:
Probably 'company_id' too as it impact the `get_base_url()` return but that's not the only place it is written without it :shrug:  so no need to r-

Reply:
it is also probably missing `website_id.domain` which is more problematic IMO